region/wreckedship/main/Bowling Alley.json

```diff
@@ -800,6 +820,112 @@
       "note": "This is a very long ceiling bomb jump.",
       "devNote": "There is no reason to go back to the left."
     },
+    {
+      "link": [2, 7],
+      "name": "G-Mode, Bowling",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": false
+        }
+      },
+      "requires": [
+        "f_DefeatedPhantoon",
+        "Morph",
+        {"or": [
+          "SpaceJump",
+          {"and": [
+            "canUseIFrames",
+            {"spikeHits": 3}
```

Comment:
Damageboosting into the ground makes 2 spike hits fully reasonable at ~expert.
You can do it without the damageboost but that has only a small number of frames lenience.